cookietemple/util/rich.py

```diff
@@ -0,0 +1,4 @@
+import sys
+from rich.console import Console
+
+console = Console(file=sys.stderr, force_terminal=True)
```

Comment:
Actually I don't know whether we should ALWAYS do this.
nf-core does this only when it detects that it is running inside Github Actions. What is the disadvantage of this? There must be one.
Maybe it breaks some terminals?

Reply:
> If Rich detects that it is not writing to a terminal it will strip control codes from the output. If you want to write control codes to a regular file then set force_terminal=True on the constructor.

From the Rich docs. I mean, we can do this only if ran inside an Action for sure, may be the safer option. But I really can't see a downside for always doing it like this. You amybe? @Zethson

Reply:
I honestly don't know what to make of this. I would suggest that we only run it inside GH Actions for now (and only then merge the PR) and we write an email/tweet William to get his opinion?

Reply:
Yes, I refactored it like this.